platforms/Windows/bld/bld.wxs

```diff
@@ -335,6 +335,12 @@
       </Component>
     </ComponentGroup>
 
+    <ComponentGroup Id="FoundationMacros" Directory="_usr_bin">
+      <Component>
+        <File Source="$(TOOLCHAIN_ROOT)\usr\bin\FoundationMacros.dll" />
```

Comment:
Is this the right path for the macro library? I'd expect it to be under a `host\plugins` folder somewhere, but if this is where the rest of the stdlib macros already live then that's fine I suppose assuming the compiler knows how to find this folder to add to `-plugin-path`

Reply:
The DLLs are in "bin", "lib" is where the import library lives. The plugin path makes sense, but I'm mirroring the path where the standard library macros are currently residing (e.g. observation macros). I think that I can do a follow up to move things around.

Reply:
Also note that #317 also put the macros into /usr/bin.